pipeline/tasks/protoc_tasks.py

```diff
@@ -359,24 +361,18 @@ def execute(self, language, api_name, output_dir, src_proto_path,
 class JavaGrpcPackmanTask(GrpcPackmanTask):
 
     def execute(self, language, api_name, output_dir, src_proto_path,
-                import_proto_path, packman_flags=None, repo_dir=None):
-        packman_flags = packman_flags or ['--experimental_alt_java']
+                import_proto_path, packman_flags=[], repo_dir=None,
+                proto_gen_pkg_deps=[]):
+        if len(packman_flags) == 0:
+            packman_flags.append('--experimental_alt_java')
```

Comment:
This can be problematic. Consider specifying `packman_flags=None`, and initialize as `packman_flags = [] if not packman_flags` at the beginning of the method.

In Python, the default value will be instantiated once and reused. And append/extend on a list is a destructive operations, so the default value itself (for further invocations) will be updated. For example:
```python
def foo(x, y=[]):
    y.append(x)
    return y

foo(1)  # => [1]
foo(2)  # => [1, 2]
```
See also: http://docs.python-guide.org/en/latest/writing/gotchas/